include/oneapi/tbb/task_group.h

```diff
@@ -199,7 +199,9 @@ class task_group_context : no_copy {
         bool reserved5          : 1;
     } my_traits;
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 1070
```

Comment:
Using this macro all over the place in our public headers is not acceptable. First it looks kind of ugly, and second the platform is not fully supported. We need to find solutions case by case to avoid those changes. What's the problem here? Is the `bit_field` attribute not supported? This structure should fit into one byte even on 32bit systems.

Reply:
@reble It is not obvious what causes the failure, but it fails:
```
In file included from /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_onetbb/onetbb/work/oneTBB-2021.5.0/src/tbb/scheduler_common.h:24,
                 from /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_onetbb/onetbb/work/oneTBB-2021.5.0/src/tbb/task_dispatcher.h:24,
                 from /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_onetbb/onetbb/work/oneTBB-2021.5.0/src/tbb/arena.cpp:17:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_devel_onetbb/onetbb/work/oneTBB-2021.5.0/src/tbb/../../include/oneapi/tbb/task_group.h:206:42: error: static assertion failed: Traits shall fit into one byte.
  206 |     static_assert(sizeof(context_traits) == 1, "Traits shall fit into one byte.");
      |                   ~~~~~~~~~~~~~~~~~~~~~~~^~~~
compilation terminated due to -Wfatal-errors.
```
I will check on ppc64 a bit later, need to get to another machine.

Reply:
@reble Apparently it was failing because size of bool on Darwin PPC is 32 bit: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=107590

Macro dropped, setting from 1 to 4 fixes static assert.